### ISSUE TITLE
Input loses focus on initially opened element

### ISSUE THREAD

**Issue by kentor** [nkbt/react-collapse#60]
Triggering a rerender with an input `onChange` handler will make the input lose focus. Subsequent changes won't. Repro here: https://jsbin.com/ziquwaragi/1/edit?js,output

On the first render there's no `<ReactMotion>` wrapping the div. On the second render there is `<ReactMotion>` wrapping the div, so react does not reuse the `<div>` dom node from the previous render containing the input, so the input loses focus.

**Comment by nkbt** [comment]
Yeah... Similar reason for this as in https://github.com/nkbt/react-collapse/issues/59
Not really sure how this one can be solved though =(. Yet

**Comment by nkbt** [comment]
Should be fixed by #67.

Please, reopen, if it is still the case with the new version 

```sh
21:55 $ npm publish

> react-collapse@2.3.2 prepublish /Users/nkbt/nkbt/react-collapse
> parallelshell -w "npm run build:lib -s" "npm run build:dist -s" "npm run build:min -s"

src/Collapse.js -> lib/Collapse.js
src/index.js -> lib/index.js
Hash: 7b89e5853699a6b94ec2
Version: webpack 1.13.1
Time: 4398ms
                Asset     Size  Chunks             Chunk Names
    react-collapse.js  17.1 kB       0  [emitted]  main
react-collapse.js.map  21.4 kB       0  [emitted]  main
    + 8 hidden modules
Hash: 9c39044d59872cb39315
Version: webpack 1.13.1
Time: 4626ms
                    Asset     Size  Chunks             Chunk Names
    react-collapse.min.js  5.83 kB       0  [emitted]  main
react-collapse.min.js.map  40.9 kB       0  [emitted]  main
    + 8 hidden modules
+ react-collapse@2.3.2
```

**Comment by nkbt** [comment]
Checked on jsbin with new version, looks good

https://jsbin.com/covalifuku/edit?html,js,console,output
